README.md

```diff
@@ -91,11 +91,35 @@ Superset provides:
 * Fast loading dashboards with configurable caching
 
 
-## Database Support
+## Supported Databases
 
 Superset speaks many SQL dialects through SQLAlchemy - a Python
-SQL toolkit that is compatible with most databases. A list of
-supported databases can be found
+SQL toolkit that is compatible with most databases. Here are some of the major database solutions that are supported:
+
+<p float="left">
```

Comment:
```suggestion
<p align="center">
```

This is totally optional, but do you think aligning images to the center would make them look better?

### Before

![image](https://user-images.githubusercontent.com/335541/88840337-289c0c80-d191-11ea-83e6-d065afb52933.png)

### After

![image](https://user-images.githubusercontent.com/335541/88840301-1c17b400-d191-11ea-8459-b8751d6f0206.png)

Reply:
Yes! 100%